CHANGES.txt

```diff
@@ -8,6 +8,9 @@ Note that build 228 was the last version supporting Python 2.
 
 Since build 302:
 ----------------
+* Tweaks to how DLLs are loaded and our installation found, which should
+  improve virtualenv support and version mismatch issues (#1787, #1794)
```

Comment:
Using `LOAD_LIBRARY_SEARCH_DEFAULT_DIRS` and `LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR` also causes other changes when loading DLL. For example, DLLs in the current directory are no longer loaded. It will be better if the change log can explain how "tweaks" are made. 😉